dash-pipeline/SAI/templates/saiapi.h.j2

```diff
@@ -381,6 +381,33 @@ typedef sai_status_t (*sai_bulk_remove_{{ table.name }}_fn)(
         _In_ sai_bulk_op_error_mode_t mode,
         _Out_ sai_status_t *object_statuses);
 
+{% endif %}
+{% if table.enable_query == 'true' %}
+/**
+ * @brief Bulk query {{ sai_api.app_name }}_{{ table.name }}
+ *
+ * @param[in] object_count Number of objects to create
+ * @param[in] {{ table.name }} List of object to create
+ * @param[in] attr_count List of attr_count. Caller passes the number
+ *    of attribute for each object to create.
+ * @param[in] attr_list List of attributes for every object.
+ * @param[in] mode Bulk operation error handling mode.
+ * @param[out] object_statuses List of status for every object. Caller needs to
```

Comment:
since bulk get API is essentially a filter-based query, hence "object_statuses" is not intuitive to add here, so do you mind to add some comment here on how we will use it (in which case, we will get failure). it will be very helpful.